qt_gui/src/qt_gui/about_handler.py

```diff
@@ -53,7 +53,7 @@ def __init__(self, qtgui_path, parent=None):
     def show(self):
         try:
             # append folder of 'qt_gui_cpp/lib' to module search path
-            qt_gui_cpp_path = os.path.realpath(get_package_path('qt_gui'))
+            qt_gui_cpp_path = os.path.realpath(get_package_path('qt_gui_cpp'))
```

Comment:
This should probably be in a separate PR. Not a big deal but something to aim for in the future.

Reply:
I don't have a problem to include this in the same PR since everything in this patch addresses feedback from another PR.

The misleading part is the title of the PR. It implies that only test related stuff is part of this patch. And when it gets squash merged that becomes the commit message. So it will be impossible to tell when looking at the commits that this little change is also part of it. And for long term maintainability these kind of things matter since they will make various tasks harder /  more effort in the future (if we have to go back through the history of the repo).